loadbalancer/fadein_test.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	fadeInDuration    = 100 * time.Millisecond
+	fadeInDuration    = 500 * time.Millisecond
```

Comment:
This is a rather specific change, why is 500ms better than 100ms here? Is this for visualization reasons? Because to me it should work independently of the `fadeInDuration` here.

Reply:
No, visualization works fine regardless of `fadeInDuration`. Such changes is needed for tests like `TestFadeIn/consistent-hash,_5` (in `loadbalancer/fadein_test.go`) because in the beginning of it there are some hosts are being "warmed up" and therefore they receive not so many requests (ones or tens) and so there're good changes that by some random reason the number of requests will go down by some little (ones or tens) amount.

Basically, this test checks that number of requests sent to every host is monotonic (decreases for old hosts, increases for fading-in hosts), so basically for fading-in hosts the condition looks like "for every period of time the number of requests in the next period of time should be no less that 0.6*number of requests in the current period". When the number of requests is small, some random reason (in the Go scheduler or random generator, for example) may easily make this condition false even for rightly implemented load balancer.

That's why I increase fadeInDuration: to increase the time window during which reqs are collected, so increase the number of requests in every specific moment of time, so decrease the influence of random events here.

Reply:
This PR tends to be too huge, moved this part to another PR: https://github.com/zalando/skipper/pull/2380